azure/hub/locals.tf

```
@@ -0,0 +1,24 @@
+locals {
+  enable_private_endpoint_subnet = var.private_endpoint_subnet != null
+
+  enable_firewall = var.firewall_config != null
+  #  firewall             = one(module.firewall)     tflint - unused
```

This will have been referred to in the outputs, set this in locals so it's with the other computed elements - will make two suggestions.
```suggestion
  firewall             = one(module.firewall)
```

---

Changed